styles.css

```diff
@@ -572,3 +556,430 @@ input:invalid {
+    padding: 0 6.5% 0 20%;
+  }
+
+  .About {
+    width: 100%;
+    height: 150vh;
+    font-weight: 400;
+    padding: none;
+    color: rgb(41, 41, 106);
+    font-size: 25px;
+    background-image: url(./image/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .my-name {
+    width: 58%;
+    height: 122px;
+    font-size: 60px;
+    align-items: center;
+    padding: 10% 6.5% 0 20%;
+    line-height: 60px;
+    margin-bottom: 0;
+  }
+
+  .contact_wraper {
+    background-image: none;
+    height: 190px;
+  }
+
+  .contact {
+    background-image: none;
+    padding-left: 20%;
+  }
+
+  ul {
+    padding: 10px;
+    margin: 0;
+  }
+
+  .connect {
+    padding-left: 13px;
+    padding-top: 10px;
+    width: 15%;
+  }
+
+  .social_media {
+    padding-top: 15px;
+    width: 300px;
+  }
+
+  .social_media_bottom img {
+    padding: 15px;
+  }
+
+  /* work section */
+  .grid-container {
+    width: 100%;
+    height: 2788px;
+    display: grid;
+    justify-items: center;
+    grid-template-rows: 2fr 2fr 2fr 2fr;
+    background-color: #e5e5e5;
+    margin-top: 50px;
+    border-top-left-radius: 50px;
+  }
+
+  .grid-container h1 {
+    display: none;
+  }
+
+  .snapshoot {
+    width: 100%;
+    height: 494px;
+    display: flex;
+    margin: 5% 5% 5% 0%;
+    background: #fff;
+    border: 6%;
+    border-radius: 24px;
+  }
+
+  .new_snapshot {
+    width: 100%;
+  }
+
+  .snapshoot_revers {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .mobile_version_img {
+    display: none;
+  }
+
+  .Mobile_vesion_list {
+    display: none;
+  }
+
+  .Desktop_vesion_list {
+    display: block;
+  }
+
+  .Desktop_vesion_img {
+    display: block;
+    padding: 24px 0 24px 24px;
+  }
+
+  .Desktop_vesion_img2 {
+    display: block;
+    padding: 24px 24px 24px 0;
+  }
+
+  .snapshoot_zero {
+    width: 100%;
+    padding: 24px 24px 24px 35px;
+  }
+
+  .snapshoot_1 ul {
+    padding: 0;
+    margin: 0;
+  }
+
+  .profile {
+    width: 296px;
+    line-height: 24px;
+    font-size: 18px;
+    font-weight: 500;
+    margin-top: 15px;
+  }
+
+  .snapshoot_1 {
+    padding-left: 14px;
+  }
+
+  .snapshoot_1 h2 {
+    padding-bottom: 5px;
+    width: 496px;
+    height: 46px;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .snapshoot p {
+    width: 496px;
+    height: 50px;
+    color: #344563;
+  }
+
+  .skill {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    margin-top: 20px;
+  }
+
+  .skill li {
+    background-color: #ebebff;
+    border-bottom: none;
+  }
+
+  /* .About_self page start  */
+  .About_self {
+    background-image: url(./image/desktop-version/about\ myself\ -\ background-\ shape-\ desktop.png);
+    background-repeat: no-repeat;
+    background-position: cover;
+    display: flex;
+    justify-content: center;
+    padding-left: 10%;
+    padding-right: 10%;
+    color: #20265a;
```

Comment:
- [ ] Another requirement that the first reviewer talked about was the fact that the spacing and alignment of the About Myself section are not identical to that of the Figma template that you chose. The space between the text and the Languages subsection is too wide. I would suggest that you reduce the gap between the two subsections. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/208254330-6a3cd8e6-0918-4781-9edd-37de290a8b48.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/208254358-2eea70b1-62bc-4a65-9006-cfceea3dddc5.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: